subsys/bluetooth/controller/ll_sw/ull_conn.c

```diff
@@ -2868,8 +2869,11 @@ static inline void event_conn_upd_init(struct ll_conn *conn,
 		}
 #endif /* CONFIG_BT_CTLR_XTAL_ADVANCED */
 
-		conn->llcp.conn_upd.pdu_win_offset = (uint16_t *)
-			&pdu_ctrl_tx->llctrl.conn_update_ind.win_offset;
+		win_offs = (void *)
+				&pdu_ctrl_tx->llctrl.conn_update_ind.win_offset;
+
+		LL_ASSERT(IS_PTR_ALIGNED(win_offs, uint16_t));
+		conn->llcp.conn_upd.pdu_win_offset = (void *)win_offs;
```

Comment:
This is intentionally misaligned as the field is inside a packed PDU struct. memcpy is used to fill this address ensuring there is no memfault.

Reply:
I assume this is true for line 3759, which is currently asserting?

Reply:
3759 in this PR?  None of the struct typecast use should assert, if they do, then its genuine misalignment/bug.

Reply:
The assert in line 3759 in ull_conn.c hits, but it's not a struct, it's an int16_t @cvinayak \

Reply:
> I assume this is true too for line 3759, which is currently asserting?

Yes.